src/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+import {createMovieList} from './pages/movieList.js';
+import {createMovieInfo} from './pages/movieDetailing.js';
+
+function routing() {
+    console.log(location);
```

Comment:
@Mmarvinn pls never put `console.log` in a PR

Reply:
fix